.gitignore

```diff
@@ -2,5 +2,6 @@
 #  if you want to ignore files created by your editor/tools,
 #  please consider a global .gitignore https://help.github.com/articles/ignoring-files
 #  please do not open a pull request to add something created by your editor or tools
-dep
+cmd/dep/dep
+cmd/dep/dep.exe
```

Comment:
The file version.go I've added was ignored by .gitignore. However @jessfraz is right. I updated .gitignore to avoid both.